platform/windows/context_gl_windows.cpp

```diff
@@ -33,6 +33,9 @@
 // Author: Juan Linietsky <reduzio@gmail.com>, (C) 2008
 
 #include "context_gl_windows.h"
+#include <dwmapi.h>
```

Comment:
Style wise, there should be an empty line before this one. See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes